webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/write/BuildBundleSequence.scala

```diff
@@ -117,14 +117,15 @@ object BuildBundleSequence {
   // (double quote is the default, and simple quote are used mostly for JSON)
   sealed trait BundleParam {
     def quote(agentEscape: String => String): String
-    def name:  String
-    def value: String
+    def name:         String
+    def value:        String
+    def variableName: Option[String]
   }
   object BundleParam       {
-    final case class SimpleQuote(value: String, name: String) extends BundleParam {
+    final case class SimpleQuote(value: String, name: String, variableName: Option[String]) extends BundleParam {
```

Comment:
why not always using it ? Is it important to use the ID on linux (I understand it is important to use the name on windows)